tests/terraform_setup/ec2/variables.tf

```diff
@@ -1,7 +1,7 @@
-variable "aws_access_key" {}
-variable "aws_secret_key" {}
+# variable "aws_access_key" {}
+# variable "aws_secret_key" {}
```

Comment:
These can be removed if I understand correctly. I am supposing this uses the role assigned to the EC2 instance where the pipeline runs.

Reply:
It uses OIDC authentication during the workflow, there's no need to pass the keys directly.